pull_request_template.md

```diff
@@ -3,9 +3,13 @@ Write a detailed description of your PR here.
 Please follow the guidelines for PRs contained [here](docs/source/contributing.md). Checklist:
```

Comment:
I also suggest replacing "PR" with "pull request" (for clarity) and moving `pull_request_template.md` to `.github` to unclutter the root folder.